content/LGinC/drone-portainer/index.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2020-07-16T00:00:00+00:00
+title: "Portainer Stack Deploy"
+author: "LGinC"
+tags: [ deploy ]
+repo: "turinge/drone-portainer"
+logo: "portainer.svg"
+image: "turinge/drone-portainer"
+---
+
+The drone plugin can be used to deploy a Docker image to a Drone environment.The below pipeline configuration demonstrates simple usage:
+
+```yaml
+steps:
+- name: portainer
+  image: turinge/drone-portainer
+  settings:
+    serverurl: http://xxxxx:9000
+    username: 
+      from_secret: portainer_username
+    password:
+      from_secret: portainer_password
+    endpointId: 1
+    stackname: xxxservice
+    imagename: xxx/xxx
+    docker_compose: |
+      version: "2"
+      services:
+        xxx:
+          image: xxx/xxx
+          ports:
+          - 80:80
+```
+# Parameter Reference
+
+serverurl
+: portainer server url. like this: http://xxx.com:9000
+
+username
+: portainer username
+
+password
+: portainer password
+
+endpointId
+: portainer endpoint id,default 1, localhost is 1 <br> portainer终结点id，默认是1,即第一个，一般为localhost
+
+stackname
+: name of stack <br> 服务栈的名称，会在stacks列表里显示
+
+imagename
+: name of pull image, like: mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine <br> 将会进行拉取镜像的镜像名
+
+docker_compose
+: content of docker-compose.yml. ps: portainer just support version: "2" <br> docker-compose.yml 的内容 注意:portainer目前仅支持version: "2" <br>
```

Comment:
Please drop or translate these Chinese strings.

Reply:
first line is translation of Chinese strings